### PR TITLE
[4.0] Standardise indents of instructions in .htaccess

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

Standardise all indents correctly - no functionality changes at all.

### Testing Instructions

Nothing to test

### Actual result BEFORE applying this Pull Request

Some instructions within modules were not indented, some were, some with space, some with tab

### Expected result AFTER applying this Pull Request

nicely laid out file 

### Documentation Changes Required

None

### CODE REVIEW

**Comment by ceford** [comment]
This looks a bit odd in comparison to other indents. Before:
```
#<IfModule mod_headers.c>
#Header always set Cross-Origin-Resource-Policy "same-origin"
#Header always set Cross-Origin-Embedder-Policy "require-corp"
#</IfModule>
```
After:
```
#<IfModule mod_headers.c>
#   Header always set Cross-Origin-Resource-Policy "same-origin"
#   Header always set Cross-Origin-Embedder-Policy "require-corp"
#</IfModule>
```
The other indents are before the #. Otherwise it is fine.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29823">issues.joomla.org/tracker/joomla-cms/29823</a>.</sub>

**Comment by PhilETaylor** [comment]
It looks funny, until you I comment the lines to make them active and then they look perfect in the .htaccess without you having to indent them

File: htaccess.txt [hunk 1]
```diff
@@ -22,20 +22,20 @@ Options -Indexes
...
+	Header always set X-Content-Type-Options "nosniff"
 </IfModule>
 
 ## Protect against certain cross-origin requests. More information can be found here:
 ## https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP)
 ## https://web.dev/why-coop-coep/
 #<IfModule mod_headers.c>
-#Header always set Cross-Origin-Resource-Policy "same-origin"
-#Header always set Cross-Origin-Embedder-Policy "require-corp"
+#   Header always set Cross-Origin-Resource-Policy "same-origin"
```
Comment (Quy) [inline htaccess.txt]: Use tab.

Comment (PhilETaylor) [reply htaccess.txt]: I use phpStorm with EditorConfig, it should be tab, but git for some reason screws the commits sometimes!

Comment (PhilETaylor) [reply htaccess.txt]: Done.

**Comment by PhilETaylor** [comment]
Converted spaces to tabs

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 8dd2777111bc126cc032ca25f9601e696aba947c<br /><br />Tested successfully in Beta3-dev of 5 July.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29823">issues.joomla.org/tracker/joomla-cms/29823</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8dd2777111bc126cc032ca25f9601e696aba947c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29823">issues.joomla.org/tracker/joomla-cms/29823</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29823">issues.joomla.org/tracker/joomla-cms/29823</a>.</sub>

**Comment by richard67** [comment]
@PhilETaylor Any reason why this is not done on staging and later merged up? Now as it is for 4.0-dev only, a comparison between the J4 and the J3 file will show differences. Not sure if that is good or bad, but I personally won't like it.

**Comment by PhilETaylor** [comment]
Well for a start most of these lines only exist in 4.0 :-)

**Comment by richard67** [comment]
Yes, I just see.

**Comment by richard67** [comment]
Thanks!

**Pull request merged**